src/libsync/account.h

```diff
@@ -75,6 +75,9 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject {
     void setSharedThis(AccountPtr sharedThis);
     AccountPtr sharedFromThis();
 
+    /// The user that can be used in dav url
```

Comment:
please comment how this relates to the shibboleth user or the user name entered in the wizard